### PR TITLE
LPS-98417 FAKE GRADLE CACHE

### PR DESCRIPTION
**Pull request by gabrielibson**

### CODE REVIEW

**Comment by gabrielibson** [comment]
Hi @lawrence-lee, here's the project-templates-form-field updated to 7.2 version. I included some changes requested by @gamerson. 
Thanks for you guys being patient. ;-) 

Cc: @natocesarrego.

**Comment by lawrence-lee** [comment]
Just started reviewing :)

File: modules/sdk/project-templates/project-templates-form-field/src/main/resources/archetype-resources/build.gradle [hunk 1]
```diff
@@ -16,11 +16,18 @@ apply plugin: "com.liferay.plugin"
...
+	compileOnly group: "com.liferay", name: "com.liferay.frontend.js.loader.modules.extender.api"
+#end
 	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel"
 #if (${liferayVersion.startsWith("7.0")})
 	compileOnly group: "com.liferay", name: "com.liferay.dynamic.data.mapping.form.field.type"
 #end
 	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations"
+#if (${liferayVersion.startsWith("7.2")})
+
+	jsCompile group: "com.liferay", name: "com.liferay.dynamic.data.mapping.form.field.type"
```
Comment (lawrence-lee) [inline modules/sdk/project-templates/project-templates-form-field/src/main/resources/archetype-resources/build.gradle:29]: We need to add `jsCompile` to the TP list of configurations here: https://github.com/liferay/liferay-portal/blob/master/modules/sdk/gradle-plugins-target-platform/src/main/java/com/liferay/gradle/plugins/target/platform/TargetPlatformPlugin.java#L264

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
pushed here: https://github.com/lawrence-lee/liferay-portal/pull/151

**Pull request closed without merging**